zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -445,6 +445,24 @@ public void convertFromSingleResultToMultipleResultsFormat(Note note) {
                 results.add(new HashMap<>());
               }
             }
+            config.put("results", results);
+          }
+        } else if (ret == null && p.getConfig() != null) {
+          if (p.getConfig().get("graph") != null && p.getConfig().get("graph") instanceof Map
```

Comment:
can you add a comment here what this block of code is for? this way we could consider removing it when we remove support for 0.6 format perhaps?